apps/docs/stories/components/Checkout/Docs.mdx

```diff
@@ -38,12 +38,26 @@ You will need to first create a checkout via [Checkout API](https://docs.justifi
 - [Security](#security)
 - [Example usage](#example-usage)
 
+- [Payment methods](#payment-methods)
+
 # Props, events and methods
 
 ---
 
 <ArgTypes exclude={['Theme']} />
 
+# Payment methods
+
+---
+
+The Unified Checkout automatically displays additional payment options when they are enabled in your account settings and when device/browser or other eligibility constraints are met:
```

Comment:
Can you please change this to: 
"...automatically displays additional payment method options..."